queue/src/main/java/com/bazaarvoice/emodb/queue/core/kafka/KafkaProducerService.java

```diff
@@ -0,0 +1,73 @@
+package com.bazaarvoice.emodb.queue.core.kafka;
+
+import org.apache.kafka.clients.producer.KafkaProducer;
+import org.apache.kafka.clients.producer.ProducerRecord;
+import org.apache.kafka.clients.producer.RecordMetadata;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Collection;
+import java.util.concurrent.Future;
+
+public class KafkaProducerService {
+    private static final Logger _log = LoggerFactory.getLogger(KafkaProducerService.class);
+    private final KafkaProducer<String, String> producer; // Changed to String
+
+    public KafkaProducerService() {
+        this.producer = new KafkaProducer<>(KafkaConfig.getProducerProps());
+        _log.info("KafkaProducerService initialized with producer properties: {}", KafkaConfig.getProducerProps());
+    }
+
+    /**
+     * Sends each message from the collection to the specified Kafka topic separately.
+     *
+     * @param topic   The Kafka topic.
+     * @param events  The collection of messages to be sent.
+     */
+    public void sendMessages(String topic, Collection<String> events, String queueType) {
+        _log.info("Sending {} messages to topic '{}'", events.size(), topic);
+        for (String event : events) {
+            _log.debug("Sending message: {}", event);
```

Comment:
That debug log will generate a huge amount of logs! Please make sure to remove before going LIVE, especially with 3 already distinct logged information per message in the `sendMessage` function.

Reply:
yes we added these debug logs for checking the end to end flow, while deploying in cert
removed the excessive logs and kept only the error logs